README.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My-Portfolio
+
+> One paragraph statement about the project.
+
+![screenshot](static/images/my-portfolio-1.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- HTML
+- CSS
+
+
+## Live Demo
+
+[Live Demo Link](https://github.com/gabrielcoder247/my-portfolio/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```

Comment:
- Please, give these sections of the readme more descriptive content. That way your readme will look more professional.